cookbooks/stac-zarr-report/data-producers/index.qmd

```diff
@@ -0,0 +1,184 @@
+
+## What are your highest priorities?
+
+It's usually not possible to optimize for all benefits at the same time. That's why we recommend picking out a few of your highest priorities. Examples include:
+    
+    1. Streamline variable-level discovery for web-based users.
+
+    2. Integrate with existing tools and workflows.
+    
+    3. Enable the simplest possible access patterns for users from `<insert-programming-language>`
+
+    4. Enable web-based visualization of large-scale multi-dimensional datasets.
+
+    5. Minimize the number of GET requests for accessing large-scale multi-dimensional datasets.
+
+    6. Minimize data transfer for accessing subsets of large-scale multi-dimensional datasets.
+
+    7. Minimize the amount of infrastructure required to maintain the catalog.
+
+    8. Minimize the cost/time of generating the catalog.
+
+    9. Limit who can read from the dataset without limiting who can inspect the metadata.
+
+For each of these examples, prioritizing one over the other would impact how much effort you dedicate to abstracting metadata into the STAC catalog. Your choices will also impact the number of GET requests needed for common access patterns and the amount of data transferred.
+
+## Options
+
+So given your priorities how should you structure the division between what metadata belongs in STAC and what belongs in Zarr?
+
+### One big Zarr store - minimal STAC metadata
+ 
+1. Create a Zarr store for each dataset (i.e., set of data collected using the same platform, algorithms, model, etc)
+2. Define a STAC collection for each Zarr dataset.
+3. In each collection include a collection-level asset containing the href link to the Zarr store.
+
+**Pros**
+- There is no metadata duplication so the STAC side is easy to maintain.
+- Simple access interface for Python users - no client-side concatenation.
+
+**Cons**
+- Potentially many GETs to construct the data cube if there is no consolidated metadata file.
+- Data variables are not exposed at the STAC level, so users cannot discover relevant datasets by searching for variables.
+
+### One big Zarr store - some STAC metadata
+
+Same as above but with the additional step:
+
+4. Write variable metadata about the groups up into the STAC collection as collection-level item-assets and summaries.
+
+**Pros**
+- Data variables are exposed at the STAC level, so users can discover relevant datasets by searching for variables.
+- Simple access interface for Python users - no client-side concatenation.
+
+**Cons**
+- Potentially many GETs to construct the data cube if there is no consolidated metadata file.
+- Metadata is duplicated, but it is variable-level so unlikely to change often.
+
+### Many smaller Zarr stores - minimal STAC metadata
```

Comment:
To check my understanding, this is _roughly_ what ESA is doing with sentinel-2, right? (modulo the silly zip-the-zarr stuff)

Reply:
yes exactly! I am going through this today to link out to all the work that the ESA et al have been doing in this space.